### DISCUSSION TITLE
Range query against `age_rating` doesn't work as expected

### DISCUSSION THREAD

**Discussion by briandking** [gotson/komga#1594] [Ideas]
### Steps to reproduce

1. Tag a series with an age_rating of 5 in Komga
2. Search for age_rating:5 to verify it works
3. Search for age_rating:[40 TO 50]  and the series with age_rating:5 will show up in the results

### Expected behavior

Only comics with an age rating falling inside the numeric range should be shown.

### Actual behavior

The series with an age rating of 5 shows up in a range search for [40 TO 50]

### Komga version

0.155.0

### Operating system

Raspbian

### Other details

Range Searches: https://komga.org/guides/search.html#:~:text=You%20can%20perform-,range%20queries,-using%20the%20%5Ba

Age rating field: https://komga.org/guides/search.html#:~:text=Example%3A%20reading_direction%3Aright_to_left-,age_rating,-.%20Example%3A%20age_rating

Location to set the age rating on a series: https://komga.org/guides/edit-metadata.html#edit-screen-details

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open issue.
- [X] I have written a short but informative title.
- [X] I have checked the [FAQ](https://komga.org/faq/).
- [X] I have updated the app to the latest version.
- [X] I will fill out all of the requested information in this form.

**Comment by gotson** [comment]
> "Note that the matching is lexicographic, not numerical"
 https://komga.org/guides/search.html#:~:text=Note%20that%20the%20matching%20is%20lexicographic%2C%20not%20numerical
